alvr/server_openvr/src/lib.rs

```diff
@@ -475,7 +475,8 @@ pub unsafe extern "C" fn HmdDriverFactory(
         let (context, events_receiver) = ServerCoreContext::new();
 
         *SERVER_CORE_CONTEXT.write() = Some(context);
-        *EVENTS_RECEIVER.lock() = Some(events_receiver);
+
+        event_loop(events_receiver);
```

Comment:
we can just let the thread dangle without keeping track of the object, that's fine with rust?

Reply:
it's fine, as long as it terminates in a reasonable time on shutdown. The problem is that we don't have a main thread to join the thread into, we only have the entry point HmdDriverFactory which exits after initialization.